lib/Data/Database.php

```diff
@@ -719,17 +771,18 @@ private static function _createPasteTable()
         list($main_key, $after_key) = self::_getPrimaryKeyClauses();
         $dataType                   = self::_getDataType();
         $attachmentType             = self::_getAttachmentType();
+        $metaType                   = self::_getMetaType();
         self::$_db->exec(
-            'CREATE TABLE ' . self::_sanitizeIdentifier('paste') . ' ( ' .
-            "dataid CHAR(16) NOT NULL$main_key, " .
-            "data $attachmentType, " .
-            'postdate INT, ' .
-            'expiredate INT, ' .
-            'opendiscussion INT, ' .
-            'burnafterreading INT, ' .
-            'meta TEXT, ' .
-            "attachment $attachmentType, " .
-            "attachmentname $dataType$after_key );"
+            'CREATE TABLE "' . self::_sanitizeIdentifier('paste') . '" ( ' .
```

Comment:
BTW, as said, in the original PR, I actually liked the ` ->bindParam` way as you could explicitly specify the data type and it all seems a lot cleaner. So I would have nothing against using that way…

Reply:
Ahhh it actually does that in that function, now I understand that… :upside_down_face:

Reply:
It's now done that way for all PDO drivers.